fiftyone/utils/transformers.py

```
@@ -1643,6 +1713,145 @@ def __call__(
+        super().__init__(classes=classes, no_background_cls=no_background_cls)
+        self._processor = None
+
+    @property
+    def processor(self):
+        return self._processor
+
+    @processor.setter
+    def processor(self, value):
+        self._processor = value
+
+    def __call__(
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__call__`

(ANN204)